cm-mlops/script/app-mlperf-inference/_cm.yaml

```diff
@@ -540,6 +540,8 @@ variations:
 
   tvm-pytorch:
     group: backend
+    base:
+      - batch_size.32
```

Comment:
I see that current batch size mechanism is not working as expected in app-mlperf-inference for tvm. `batch_size.32` in base variation hardcodes the batchsize to 32 and a user won't be able to change it as he desires. Ideally we would like to make 32 as the default batchsize and then user can override it like he wants. If you think that makes sense we can discuss how to do it in a proper way. 
@gfursin

Reply:
Thanks for the fix. We'll make the batchsize option work as expected soon.